log/lvl.go

```diff
@@ -106,12 +112,10 @@ func lvl(lvl, skip int, args ...interface{}) {
 		}
 
 		namePadding := 0
-		linePadding := 0
```

Comment:
Why remove the line padding?

Reply:
If you don't want it for the `AbsoluteFilePath=true`, then just set it to `0` in that code-path. But else it gives a nicer formatted output. 

Yes, I'm being anal about this, but when looking at 10'000 lines of debug-output from Travis, I _love_ having all those outputs nicely aligned. It speeds up my detection rate by a factor of 2. No joke for once! Please keep the padding also for the lines.

Reply:
I was still working on the formatting but the final solution looks like this:
```
1 : /home/gaylor/go/src/go.dedis.ch/onet/server.go:217(onet.(*Server).Start)       - Starting server at 2019-10-09 10:59:49 on address tcp://127.0.0.1:46795
2 : /home/gaylor/go/src/go.dedis.ch/onet/websocket.go:186(onet.(*WebSocket).start) - Starting to listen on 0.0.0.0:46796
E : /home/gaylor/go/src/go.dedis.ch/cothority/blscosi/api_test.go:21(blscosi.TestClient_SignatureRequest) - Got an empty roster-list
1 : /home/gaylor/go/src/go.dedis.ch/cothority/blscosi/api_test.go:26(blscosi.TestClient_SignatureRequest) - Sending request to service... tcp://127.0.0.1:41339
2 : /home/gaylor/go/src/go.dedis.ch/onet/websocket.go:280(onet.wsHandler.ServeHTTP)                       - ws request from [::1]:55134: blsCoSiService/SignatureRequest
E : /home/gaylor/go/src/go.dedis.ch/onet/websocket.go:333(onet.wsHandler.ServeHTTP)                       - Got an error while executing blsCoSiService/SignatureRequest: response: test: root:
```

Reply:
OK - so you don't need the line-number, because it's already in the fullpath. But what if the fullpath is `false`? Then you won't have the line-numbers, no?

Also, I'd put a space between the full path and the method-name for readability.

Reply:
fullpath to false only means you get the file name + line number to avoid printing the absolute path. The line number is appended so IDEs can bring you to the line.

Reply:
what I mean: if `fullpath == false`, then the log-output only contains the method-name, but not the line number. If this is correct, then I'd like to have the line-number back ;)

Reply:
Yes you still have the filename with the number, only what is before is not printed.

Reply:
Oh my - took me ages to understand that `if`...